Avoid
    error: ordered comparison between pointer and zero ('const void *' and 'int')

Patch taken from FreeBSD (https://svnweb.freebsd.org/ports?view=revision&revision=487863).

--- qtiplot/src/core/ApplicationWindow.cpp.orig	2019-08-20 16:39:14.000000000 -0700
+++ qtiplot/src/core/ApplicationWindow.cpp	2019-08-20 16:42:40.000000000 -0700
@@ -10933,11 +10933,11 @@
 			foreach(Graph *g, layers){
 				QStringList onPlot = g->curveNamesList();
 				onPlot = onPlot.grep (name,TRUE);
-				if (int(onPlot.count()) && plots.contains(w->objectName())<=0)
+				if (onPlot.count() && !plots.contains(w->objectName()))
 					plots << w->objectName();
 			}
 		}else if (w->isA("Graph3D")){
-			if ((((Graph3D*)w)->formula()).contains(name,TRUE) && plots.contains(w->objectName())<=0)
+			if ((((Graph3D*)w)->formula()).contains(name,TRUE) && !plots.contains(w->objectName()))
 				plots << w->objectName();
 		}
 	}
@@ -10954,7 +10954,7 @@
 		for (int j=0; j<onPlot.count(); j++)
 		{
 			QStringList tl = onPlot[j].split("_", QString::SkipEmptyParts);
-			if (tables.contains(tl[0])<=0)
+			if (!tables.contains(tl[0]))
 				tables << tl[0];
 		}
 	}
--- qtiplot/src/plot3D/Graph3D.cpp.orig	2011-08-24 12:25:12.000000000 -0700
+++ qtiplot/src/plot3D/Graph3D.cpp	2019-08-20 16:43:59.000000000 -0700
@@ -3283,7 +3283,7 @@
 				return 0;
 			plot->addRibbon(t, l[0], l[1], fList[2].toDouble(), fList[3].toDouble(),
 					fList[4].toDouble(), fList[5].toDouble(), fList[6].toDouble(), fList[7].toDouble());
-		} else if (formula.contains("(Z)",true) > 0){
+		} else if (formula.contains("(Z)",true)){
 			formula.remove("(X)").remove("(Y)").remove("(Z)");
 			QStringList l = formula.split(",");
 			if (l.size() < 3)
--- qtiplot/src/table/Table.cpp.orig	2011-09-13 13:20:30.000000000 -0700
+++ qtiplot/src/table/Table.cpp	2019-08-20 16:37:54.000000000 -0700
@@ -1080,7 +1080,7 @@
 		if (enumerateRight)
             newLabel += QString::number(n);
 
-		if (col_label.contains(newLabel) > 0){
+		if (col_label.contains(newLabel)){
 			if (warn){
 				QMessageBox::critical(0, tr("QtiPlot - Error"),
 				tr("There is already a column called : <b>"+newLabel+"</b> in table <b>"+caption+"</b>!<p>Please choose another name!"));
